src/components/LoginSignup/LogIn.js

```diff
@@ -0,0 +1,136 @@
+import React from "react";
+import { useState } from "react";
+import { navigate } from "@reach/router";
+import { signIn } from "../../auth/signIn";
+import { makeStyles } from "@material-ui/core/styles";
+import { TextField, Button, Snackbar, Slide } from "@material-ui/core/";
+import MuiAlert from "@material-ui/lab/Alert";
+import { Wrapper } from "./Wrapper";
+
+const useStyles = makeStyles((theme) => ({
+  textField: {
+    marginBottom: theme.spacing(2),
+    "& label.Mui-focused": {
+      color: theme.palette.grey[900],
+    },
+  },
+  button: {
+    marginTop: theme.spacing(2),
+    color: theme.palette.grey[900],
+  },
+  link: {
+    textDecoration: "none",
+    color: theme.palette.grey[900],
+  },
+  errorMessage: {
+    marginBottom: theme.spacing(2),
+    color: theme.palette.error.main,
+  },
+}));
+
+function SignIn({ location }) {
+  const classes = useStyles();
+  const [emailValue, setEmailValue] = useState("");
+  const [passwordValue, setPasswordValue] = useState("");
+  const [errorMessage, setErrorMessage] = useState("");
+  const [open, setOpen] = useState(true);
+  let signUpSuccessful = false;
+
+  try {
+    if (location.state.signUpSuccessful) {
+      signUpSuccessful = true;
+    }
+  } catch {}
+
+  const onSignInClicked = async () => {
+    try {
+      const signInSuccessful = await signIn(emailValue, passwordValue);
+      signInSuccessful.successful && navigate(`/`);
+      switch (signInSuccessful.error) {
+        case "auth/invalid-email":
+          setErrorMessage("Die angegebene E-Mail Adresse ist falsch formatiert.");
+          break;
+        case "auth/wrong-password":
+          setErrorMessage("Das angegebene Passwort ist falsch.");
+          break;
+        case "auth/user-not-found":
+          setErrorMessage(
+            "Der Account existiert leider nicht. Bitte überprüfe deine Eingabe oder registriere dich."
+          );
+          break;
+        default:
+          setErrorMessage(
+            "Leider ist ein Fehler aufgetreten. Bitte überprüfe deine Eingabe oder probiere es später noch einmal."
+          );
+      }
+    } catch (e) {
+      throw new Error("Signing in failed. Please try again later.");
+    }
+  };
+
+  return (
+    <>
+      {signUpSuccessful && (
+        <Snackbar
+          open={open}
+          autoHideDuration={7000}
+          onClose={() => setOpen(false)}
+          anchorOrigin={{ vertical: "top", horizontal: "center" }}
+          TransitionComponent={Slide}
+        >
+          <MuiAlert
+            elevation={6}
+            variant="filled"
+            severity="success"
+            onClose={() => setOpen(false)}
+          >
+            Deine Registrierung war erfolgreich 🎉. <br /> Du kannst dich nun einloggen.
```

Comment:
```suggestion
            Deine Registrierung war erfolgreich 🎉 <br /> Du kannst dich nun einloggen.
```
Ohne den Punkt sieht es besser aus 😄